cost/turbonomics/rightsize_databases_recommendations/aws/turbonomics_rightsize_databases_recommendations.pt

```diff
@@ -311,6 +446,87 @@ policy "pol_turbonomic_rightsize_databases" do
       field "createdTime" do
         label "Created Time"
```

Comment:
Check if it is database created time or action created time in the response. If it is action created time, change the name of the label to "Action Created Time".

Reply:
Hi Vic, since is the only date from the JSON response, i believe this is the date when the database server was created